RootTools/python/samples/autoAAAconfig.py

```diff
@@ -9,7 +9,7 @@ def autoAAA(selectedComponents):
         if not re.match("/[^/]+/[^/]+/MINIAOD(SIM)?", comp.dataset): continue
         if "/store/" not in comp.files[0]: continue
         if re.search("/store/(group|user|cmst3)/", comp.files[0]): continue
-        if comp.isData and "PromptReco" in comp.dataset: continue
```

Comment:
go back to the original version. If we need to change this we need a different PR, separated from MonoXAnalysis